config/routes.js

```diff
@@ -264,7 +262,14 @@ module.exports = function (app) {
   bgm.resubscribePendingJobs();
 
   // API ROUTES
-  //app.post("/api/v1/submit", api.apiSubmit);
-  app.post("/api/v1/submit", api.apiSubmit);
-  app.post("/api/v1/debug", slac.invokeDEBUG);
+  app.post("/api/v1/submit", api.checkAPIKey, api.apiSubmit);
+  app.get("/api/v1/status", api.apiStatus);
+
+  // API KEY ROUTES
+  api_verify = require(path.join(__dirname, "../app/routes/api"));
+  app.get("/apikey", api_verify.renderApi);
+  app.post("/api/v1/issueKey", api.checkCapcha, api.issueKey);
```

Comment:
Can't there be a way to bypass checkCapcha for dev purposes?

Reply:
Dev mode toggle has now been added to the config file. It allows you to bypasses the API key check when making requests.